__tests__/components/Form.spec.js

```diff
@@ -405,4 +405,31 @@ describe('Form component', () => {
       component.unmount()
     })
   })
+
+  describe('patternProperties', () => {
+    it('should allow you to add new key value pairs', () => {
+      const schema = {
+        type: 'object',
+        patternProperties: {
+          '^[0-9]+$': {
+            'title': 'Rule',
+            'type': 'string'
+          }
+        }
+      }
+
+      const component = mount(
+        <Provider>
+          <Form
+            schema={schema}
+          />
+        </Provider>
+      )
+
+      const valueInput = component.find('input.rendition-form-pattern-properties__value-field')
```

Comment:
would having an additional test, that the object produced from the form is in the right shape, help ?

Reply:
I spent some time wrestling with Enzyme trying to get tests to run correctly, but kept encountering a very strange bug. In the end I decided to continue with the PR and come back to unit tests as I didn't want to block and alpha release of this functionality.

Reply:
alrighty then :)